scripts/e2etests/VSUtils.ps1

```diff
@@ -207,6 +207,42 @@ function Update-Configuration(
     Start-Process -FilePath $vsFilePath -ArgumentList '/updateConfiguration' -Wait
 }
 
+function UpdateVSInstaller {
+    param(
+        [ValidateSet("16.0")]
+        [string]$VSVersion,
+        [Parameter(Mandatory = $true)]
+        [int]$ProcessExitTimeoutInSeconds
+    )
+
+    $vsMajorVersion = [System.Version]::Parse($VSVersion).Major
+    $vsBootstrapperUrl = "https://aka.ms/vs/$vsMajorVersion/pre/vs_enterprise.exe"
+
+    $tempdir = [System.IO.Path]::GetTempPath()
+    $VSBootstrapperPath =  "$tempdir" + "vs_enterprise.exe "
```

Comment:
Just a note: I see that Tyler's suggestion was used and your [comment](https://github.com/NuGet/NuGet.Client/pull/3209/files#r402772537) in the other PR no longer applies

Reply:
Yes, my previous comment was based on the doc. But Typer replied that,	`I don’t think there is a vs_installer.exe aka.ms link. I think that may be an issue in the doc. `